lib/js/jsre.nim

```diff
@@ -44,3 +45,7 @@ func test*(self: RegExp; pattern: cstring): bool {.importjs: "#.test(#)".}
 
 func toString*(self: RegExp): cstring {.importjs: "#.toString()".}
   ## Returns a string representing the RegExp object.
+
```

Comment:
how about making the API more complete by adding wrappers for `matchAll` + friends:

> These patterns are used with the exec() and test() methods of RegExp, and with the match(), matchAll(), replace(), replaceAll(), search(), and split() methods of String.

(https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions)

Reply:
Ye